docs/vm-images.md

```diff
@@ -166,45 +206,63 @@ a description of our support for Windows images.
 
 <sup><b>*</b></sup> Chrome Remote desktop does not support Ubuntu 20.04, but it does support Ubuntu 22.04.
 
-## Other Images
+### Other Images
 
 The HPC Toolkit strives to provide flexibility wherever possible. It is possible
 to set a VM image in many HPC Toolkit modules. While we do not officially
 support images not listed here, other public and custom images should work with
 the majority of modules with or without further customization, such as custom
 startup-scripts.
 
-## Slurm on GCP Custom Images
+## Slurm on GCP
+
+### Publicly Published Slurm Images
+
+SchedMD publishes "Slurm on GCP" public images, which are documented
+[here][slurm-gcp-images]. This documentation covers which images are available
+and what software is installed on them.
+
+> **_Note:_** When deploying a Slurm cluster, the HPC Toolkit Terraform wrapper
+modules enforce that specific image versions are used, to ensure compatibility.
+Currently, the **slurm-gcp-5-7** image families are enforced for the Slurm v5
```

Comment:
I would rather not name a specific release in the text especially as this only matches the beginning of each families name. Suggest removing the line in favor of:

```
Slurm images are compatible by the minor version releases of the Terraform and Packer modules. For example, images built for version 5.8 are compatible with all 5.8 Terraform modules beginning with 5.8.0. The version of the Slurm modules used by your copy of the Toolkit in the local filesystem can be inspected by looking for the `source` line in community/modules/compute/schedmd-slurm-gcp-v5-partition/main.tf.
```

The latest GitHub release supports [these images](https://github.com/SchedMD/slurm-gcp/blob/5.7.6/docs/images.md#published-image-family).

@cboneti This is definitely open for some thought.

Reply:
Resolved.